pkg/validation/external/removed_apis.go

```diff
@@ -0,0 +1,43 @@
+package external
+
+import (
+	"github.com/operator-framework/api/pkg/manifests"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func GetRemovedAPIsOn1_25From(bundle *manifests.Bundle) map[string][]string {
```

Comment:
Why not add this implementation with https://github.com/operator-framework/api/blob/master/pkg/validation/internal/removed_apis.go?